#!/usr/bin/env python
PACKAGE = "takin_motors"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

brake_enum = gen.enum([gen.const("Brake",   int_t, 0, "Brake mode"),
                       gen.const("Coast",   int_t, 1, "Coast mode")],
                     "brake mode")

gen.add("brake_mode", int_t, 0, "Motors brake mode", 0, 0, 1, edit_method=brake_enum)

exit(gen.generate(PACKAGE, "takin_motors", "Motor"))